pos_order_to_sale_order/readme/newsfragments/2928.bigfix.rst

```diff
@@ -0,0 +1 @@
+Sale order line description computed by default with adding Customer note.
```

Comment:
This is not a bug. A feature is being added.
Add readable and clear feature description please